src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -228,7 +228,7 @@ SELECT * FROM pg_dist_shard ORDER BY shardid;
  mx_testing_schema.mx_test_table | 1310007 | t            | 1610612736    | 2147483647
 (8 rows)
 
-SELECT * FROM pg_dist_shard_placement ORDER BY shardid;
+SELECT * FROM pg_dist_shard_placement ORDER BY shardid, nodename, nodeport;
```

Comment:
It wouldn't make much difference here, since there is only one placement per shard here.